src/graph/graph.c

```diff
@@ -767,6 +767,9 @@ void Graph_RemoveNodeLabels
 	RG_Matrix nl = Graph_GetNodeLabelMatrix(g);
 	for(uint i = 0; i < lbl_count; i++) {
 		LabelID   l = lbls[i];
+		if(!Graph_IsNodeLabeled(g, id, l)) {
+			continue;
+		}
```

Comment:
simple enough, worth adding a comment say:
`it is possible for the node to not be associated with the current label, RG_Matrix_removeElement_BOOL always returns GrB_SUCCESS`